### DISCUSSION TITLE
Form Field Description

### DISCUSSION THREAD

**Discussion by jeffreylauwers** [nl-design-system/backlog#227] [Component suggestions]
<img width="480" alt="component_form-field-description" src="https://github.com/nl-design-system/backlog/assets/545207/6dfe6fb6-cc73-490e-814b-f8385edd9b38">

We willen graag meer leren over het Form Field Description component. In welke vorm, of vormen, wordt deze momenteel ingezet en toegepast binnen jouw organisatie?

Deel hieronder een link naar Figma, Storybook of screenshots. Met deze informatie kunnen we bewijzen dat het component onderdeel moet worden van de NL Design System component library.

Daarnaast wordt het delen van onderzoek gewaardeerd. Dus heb je zelf onderzoek gedaan en zijn er inzichten voor dit component? Dan horen we dit ook graag!

[Link naar bericht in Slack](https://codefornl.slack.com/archives/C7KDQRXMZ/p1709819672446039)

---

## Naam
Form Field Description

## Doel
Toont ondersteunende tekst en biedt extra context over de in te vullen informatie.

## Gerelateerde componenten
[Form Field](https://github.com/orgs/nl-design-system/discussions/311), [Form Field Label](https://github.com/orgs/nl-design-system/discussions/226), [Form Field Label Suffix](https://github.com/orgs/nl-design-system/discussions/416), [Form Field Error Message](https://github.com/orgs/nl-design-system/discussions/228)

## Richtlijnen

[Descriptions in een formulier - NL Design System](https://www.nldesignsystem.nl/richtlijnen/formulieren/descriptions/)
Onze richtlijnen rondom descriptions in formulieren.

[Placeholders in een formulier - NL Design System](https://www.nldesignsystem.nl/richtlijnen/formulieren/placeholders)
Onze richtlijnen rondom placeholders in formulieren.

## Onderzoek

[Avoid Messages Under Fields - Adrian Roselli](http://adrianroselli.com/2017/01/avoid-messages-under-fields.html)
Artikel beschrijft waarom Form Field Descriptions beter tussen het label en form control geplaatst kunnen worden.

[Accessible description exposure - Adrian Roselli](https://adrianroselli.com/2022/04/accessible-description-exposure.html)
Artikel waarin Adrian Roselli
 laat zien dat `aria-describedby` zeer inconsistent wordt aangekondigd door verschillende screenreaders en navigatiemethoden, waardoor je er niet op moet vertrouwen voor cruciale informatie.

[The problem with placeholders and what to do instead - Adam Silver](https://adamsilver.io/blog/the-problem-with-placeholders-and-what-to-do-instead/)
Artikel beschrijft waarom placeholders voor problemen zorgen.

[Gebruikersonderzoek Klachtenformulier Gemeente Venray - gebruikersonderzoeken.nl](https://gebruikersonderzoeken.nl/docs/onderzoek-bekijken/formulieren/venray-klachtenformulier-onderzoek/)
Ook tijdens een onderzoek bij Gemeente Venray kwam naar voren dat men beschrijvingen onder de velden over het hoofd zag.

**Comment by jeffreylauwers** [comment]
## Amsterdam

- [Storybook](https://designsystem.amsterdam/?path=/story/components-forms-field--with-description) (in combinatie met Field)
- [GitHub](https://github.com/Amsterdam/design-system/tree/develop/packages/css/src/components/field)
- [Figma](https://www.figma.com/file/9IGm6IdPUYizBNGsUnueBd/Amsterdam-Design-System?type=design&node-id=9413-20735&mode=design&t=fPx0tp2iRNhnTiiM-4) (in combinatie met Text input)

---

### Screenshots / Varianten

**Figma**
Niet beschikbaar als 'los' component maar wel toegepast in combinatie met verschillende form controls. Onder andere...

Text input
![Scherm­afbeelding 2024-03-13 om 08 28 28](https://github.com/nl-design-system/backlog/assets/545207/f51605f2-c847-4cf5-88e9-978ce823a630)

Datepicker
![Scherm­afbeelding 2024-03-13 om 08 30 35](https://github.com/nl-design-system/backlog/assets/545207/6f39d251-1be0-42fe-ae91-7b256a969b14)

**Storybook**
Niet beschikbaar als 'los' component maar wel gevonden bij story van Field (With description)

<img width="968" height="170" alt="Scherm­afbeelding 2025-12-02 om 16 03 27" src="https://github.com/user-attachments/assets/cdab99da-3d99-4a05-bc4a-1099eda8db6f" />

---

### Notities

In Figma
- Wordt 'Hint' genoemd.
- Optie om boven en onder form control (Text input) te tonen.
- Boven foutmelding.

Storybook
- Onder Field (With description).
- Is in code een paragraaf die via `aria-describedby` gekoppeld wordt aan het invulveld.

**Comment by jeffreylauwers** [comment]
## Den Haag

- [Storybook](https://nl-design-system.github.io/denhaag/?path=/docs/css-formfielddescription--docs)
- [Figma - Huidig](https://www.figma.com/file/JpoY3waVoQGlLQzQXTL9nn/%F0%9F%8C%BB-Den-Haag-Design-System?type=design&node-id=1-1653&mode=design&t=pbACC6679d52NI2X-4) (in combinatie met Text input)
- [Figma - WIP](https://www.figma.com/file/x4RkF6BIdrNZbh7D53NTzB/%F0%9F%92%A0-NLDS---Den-Haag---Bibliotheek?type=design&node-id=851-929&mode=design&t=smfzZ6f7Q0Fln7nT-4)

---

### Screenshots / Varianten

**Storybook**

Default
![Scherm­afbeelding 2024-03-13 om 08 34 37](https://github.com/nl-design-system/backlog/assets/545207/55cd0655-30aa-426d-81bd-f9a2bfd8a71e)

Invalid
![Scherm­afbeelding 2024-03-13 om 08 34 53](https://github.com/nl-design-system/backlog/assets/545207/26b3886e-254b-4c83-85ea-db40c4598e3a)

Valid
![Scherm­afbeelding 2024-03-13 om 08 34 57](https://github.com/nl-design-system/backlog/assets/545207/8aeb776d-1fb3-405c-b8b2-26a8ee0aeca3)

In combinatie met Text input
![Scherm­afbeelding 2024-03-13 om 08 46 06](https://github.com/nl-design-system/backlog/assets/545207/032c0370-5a4d-4a7f-bf61-494e0a5e44e6)


**Figma - Huidig**
Niet beschikbaar als 'los' component maar wel toegepast in combinatie met verschillende form controls. Onder andere...

Text input
![Scherm­afbeelding 2024-03-13 om 08 33 36](https://github.com/nl-design-system/backlog/assets/545207/38427dbe-36a0-49e9-b37d-7e780f686203)

Date picker
![Scherm­afbeelding 2024-03-13 om 08 33 36](https://github.com/nl-design-system/backlog/assets/545207/38427dbe-36a0-49e9-b37d-7e780f686203)

Radio group
![Scherm­afbeelding 2024-03-13 om 12 45 23](https://github.com/nl-design-system/backlog/assets/545207/10ae7a15-933a-424c-ae54-69dc5f3878f3)

Text area
![Scherm­afbeelding 2024-03-13 om 08 37 02](https://github.com/nl-design-system/backlog/assets/545207/43deba2f-d46a-4e58-857f-6ee0fa0213c0)

**Figma - WIP**
![Scherm­afbeelding 2024-03-13 om 08 37 38](https://github.com/nl-design-system/backlog/assets/545207/62e73daf-4b97-46e0-8947-8073b2ff4a82)

![Scherm­afbeelding 2024-03-13 om 08 40 11](https://github.com/nl-design-system/backlog/assets/545207/5673b048-5e0f-4955-ac25-a5409eb07192)

![Scherm­afbeelding 2024-03-13 om 08 40 15](https://github.com/nl-design-system/backlog/assets/545207/88cf8284-4cec-4521-a2a7-6828f945437f)

---

### Notities
- In Storybook ook een Invalid en Valid state. Maar ook een [Form field error message](https://nl-design-system.github.io/denhaag/?path=/docs/css-form-field-error-message--docs).
- In Storybook wordt de description boven een form control geplaatst.
- In Storybook wordt de description boven de error message geplaatst, en zit het form control er tussen.
- In Figma - Huidig wordt de description onder een form control geplaatst.
- In Figma - Huidig wordt de description gecombineerd met error message.
- In Figma - WIP wordt de description boven een form control geplaatst.
- In Figma - WIP wordt de description boven de error message geplaatst.

**Comment by jeffreylauwers** [comment]
## DSO Toolkit

[Storybook](https://storybook.dso-toolkit.nl/62.8.4/?path=/story/html-css-form-groups-input--input) (in combinatie met Text input)

---

### Screenshots / Varianten

**Storybook**
Niet beschikbaar als 'los' component maar wel toegepast in combinatie met verschillende form controls. Onder andere...

Text input
![Scherm­afbeelding 2024-03-13 om 08 50 56](https://github.com/nl-design-system/backlog/assets/545207/e7f2217f-3527-4e23-baa8-e9892c029753)

Select
![Scherm­afbeelding 2024-03-13 om 08 52 11](https://github.com/nl-design-system/backlog/assets/545207/98ff559e-996f-495a-ad57-417b22e65b23)

Date picker
![Scherm­afbeelding 2024-03-13 om 08 52 56](https://github.com/nl-design-system/backlog/assets/545207/44e7e39c-7701-453f-bc4e-567051460668)

Invalid state Text input
![Scherm­afbeelding 2024-03-13 om 08 54 41](https://github.com/nl-design-system/backlog/assets/545207/3c4b5ea8-ee01-42e4-b7cb-c74aafaf347e)


---

### Notities
- Wordt 'Helptext' genoemd.
- Description wordt onder een form control geplaatst.
- Description wordt onder de error message geplaatst.

**Comment by jeffreylauwers** [comment]
## DUO

[Documentatie website](https://uno.dfront.rijkscloud.nl/#/components/textinput) (in combinatie met Text input)

---

### Screenshots / Varianten

**Documentatie website**
Niet beschikbaar als 'los' component maar wel toegepast in combinatie met verschillende form controls. Onder andere...

Text input
![Scherm­afbeelding 2024-03-13 om 08 59 45](https://github.com/nl-design-system/backlog/assets/545207/3c262485-671d-4f18-b181-6bf62f1c72d3)

Radio group
![Scherm­afbeelding 2024-03-13 om 08 59 13](https://github.com/nl-design-system/backlog/assets/545207/87a8f6b4-0038-42c5-85c3-1d99cb06cea6)

Text area
![Scherm­afbeelding 2024-03-13 om 09 00 02](https://github.com/nl-design-system/backlog/assets/545207/d19ad1c0-bfe9-4563-a198-531749d63a7a)

Text input - Invalid state
![Scherm­afbeelding 2024-03-13 om 09 00 13](https://github.com/nl-design-system/backlog/assets/545207/026fd186-040e-468a-839c-801e3478bb04)

---

### Notities
- Wordt 'Hinttekst' genoemd.
- Description wordt direct onder het label geplaatst, niet altijd boven een form control vanwege de [optie om horizontaal uit te lijnen](https://github.com/orgs/nl-design-system/discussions/226#discussioncomment-8757819).
- Description wordt boven de error message geplaatst.

**Comment by jeffreylauwers** [comment]
## Open Formulieren

[Storybook](https://open-formulieren.github.io/open-forms-sdk/?path=/docs/pure-react-components-forms-textfield--docs) (in combinatie met Text input)

---

### Screenshots / Varianten

**Storybook**
Niet beschikbaar als 'los' component maar wel toegepast in combinatie met verschillende form controls. Onder andere...

Text input
![Scherm­afbeelding 2024-03-13 om 09 13 58](https://github.com/nl-design-system/backlog/assets/545207/bcd8e9b8-8b26-44f8-9d26-d65e6503999d)

Select
![Scherm­afbeelding 2024-03-13 om 09 14 36](https://github.com/nl-design-system/backlog/assets/545207/b0ee7ee0-4448-419f-a25c-9322b1165b02)

Date picker
![Scherm­afbeelding 2024-03-13 om 09 13 44](https://github.com/nl-design-system/backlog/assets/545207/ad7c1488-e0fd-48a3-a9a1-6ea310d8a219)

Radio group
![Scherm­afbeelding 2024-02-27 om 15 27 43](https://github.com/nl-design-system/backlog/assets/545207/4a4cc21e-864a-4de7-bc6b-6928123159df)

Text input - Invalid state
![Scherm­afbeelding 2024-03-13 om 09 15 39](https://github.com/nl-design-system/backlog/assets/545207/6018bf10-ac29-44d7-a8ec-a3ec6a9bb0e2)

---

### Notities
- Description wordt onder een form control geplaatst inclusief achtergrond kleur met border-left.
- Description wordt boven de error message geplaatst.

**Comment by jeffreylauwers** [comment]
## Open Gemeenten
[Patternlab - Forms and Search](https://patternlab.lenteveld.nl/patterns/01-molecules-080-Forms-And-Search/index.html)

---

### Screenshots / Varianten

**Patternlab**
Niet beschikbaar als 'los' component maar wel toegepast in combinatie met verschillende form controls. Onder andere...

Text input
![Scherm­afbeelding 2024-03-13 om 09 32 43](https://github.com/nl-design-system/backlog/assets/545207/9e57262e-fcf1-4f4a-93b8-b87d8698daaf)

Datepicker
![Scherm­afbeelding 2024-03-13 om 09 33 22](https://github.com/nl-design-system/backlog/assets/545207/e5922917-b665-4304-8bfa-501172ff9ef7)

Select
![Scherm­afbeelding 2024-03-13 om 09 32 53](https://github.com/nl-design-system/backlog/assets/545207/b953aa7c-e5e8-4414-b381-c3dedf4eebb6)

Text area
![Scherm­afbeelding 2024-03-13 om 09 32 22](https://github.com/nl-design-system/backlog/assets/545207/4fb2f68c-7d8c-4c46-961f-b43c0aca00ea)

Radio group
![Scherm­afbeelding 2024-03-13 om 09 33 03](https://github.com/nl-design-system/backlog/assets/545207/3d56cb76-b4a0-4806-824c-0d6b22071070)

---

### Notities
- Description wordt onder een form control geplaatst.
- Description wordt [Martijn zoekt dit uit] error message geplaatst.

**Comment by jeffreylauwers** [comment]
## RVO

- [Documentatie website](https://nl-design-system.github.io/rvo/docs/componenten/form-field-textinput/)
- [Storybook](https://nl-design-system.github.io/rvo/?path=/docs/componenten-text-input-field--documentatie)
- [Figma](https://www.figma.com/file/Sj6myBL1Fvot5M1qGxzvEo/ROOS-(RVO-Design-System)?type=design&node-id=4307-11776&mode=design&t=jxLIvX82VwfkYSF6-4)

---

### Screenshots / Varianten

**Storybook**
Niet beschikbaar als 'los' component maar wel toegepast in combinatie met verschillende form controls. Onder andere...

Text input
![Scherm­afbeelding 2024-03-13 om 10 03 22](https://github.com/nl-design-system/backlog/assets/545207/90f9e55e-1fdf-408e-b015-3b26cc7f995f)

Datepicker
![Scherm­afbeelding 2024-03-13 om 10 02 51](https://github.com/nl-design-system/backlog/assets/545207/4f3667f3-1e6e-418a-a2f6-1d555e8c07b9)

Select
![Scherm­afbeelding 2024-03-13 om 10 03 16](https://github.com/nl-design-system/backlog/assets/545207/072b9c7e-bada-404e-bbc6-8d883d9a18a1)

Radio group
![Scherm­afbeelding 2024-03-13 om 10 03 10](https://github.com/nl-design-system/backlog/assets/545207/e2b8004e-f7e2-417c-8b53-812ed5b5552e)

Text input - Invalid state
![Scherm­afbeelding 2024-03-13 om 14 06 20](https://github.com/nl-design-system/backlog/assets/545207/55f238ec-16ac-4ebb-b092-61e008483a05)


---

### Notities
- Wordt 'Helper text' genoemd.
- Description wordt boven een form control geplaatst.
- Description wordt boven error message geplaatst.

**Comment by jeffreylauwers** [comment]
## Utrecht

- [Storybook](https://nl-design-system.github.io/utrecht/storybook/?path=/docs/css_css-form-field-description--docs)
- [GitHub](https://github.com/nl-design-system/utrecht/tree/main/components/form-field-description)
- [Figma](https://www.figma.com/file/UXIHcIurAD8hyoBWx4hDBV/NLDS---Gemeente-Utrecht---Bibliotheek?type=design&node-id=851-929&mode=design&t=0NpXKky0keqojruQ-4)

---

### Screenshots / Varianten

**Storybook**

Default
![Scherm­afbeelding 2024-03-13 om 10 28 33](https://github.com/nl-design-system/backlog/assets/545207/22b861f4-f1e3-4e33-ae07-947c8d171686)

Invalid
![Scherm­afbeelding 2024-03-13 om 10 28 37](https://github.com/nl-design-system/backlog/assets/545207/230d8ff8-e607-4e74-815e-23483445747e)

Valid
![Scherm­afbeelding 2024-03-13 om 10 28 41](https://github.com/nl-design-system/backlog/assets/545207/10f713af-b859-4fc1-97dd-635d8b387aa9)

In combinatie met Text input
![Scherm­afbeelding 2024-03-13 om 10 33 07](https://github.com/nl-design-system/backlog/assets/545207/f5444ed5-21ad-41db-afb2-ceb0645a29fb)

**Figma**
![Scherm­afbeelding 2024-03-13 om 10 33 42](https://github.com/nl-design-system/backlog/assets/545207/eb412093-c49f-49b7-ab53-63ee4cd7129e)

![Scherm­afbeelding 2024-03-13 om 08 42 46](https://github.com/nl-design-system/backlog/assets/545207/382d7e56-0a6a-472c-939e-959b51d53838)

![Scherm­afbeelding 2024-03-13 om 08 42 52](https://github.com/nl-design-system/backlog/assets/545207/bc1bc795-857b-4623-a357-89e30e1d5481)

---

### Notities
- Uitgebreide documentatie beschikbaar.
- Description wordt boven een form control geplaatst.
- Description wordt boven error message geplaatst.

**Comment by jeffreylauwers** [comment]
## ✨ Dit component is nu Help Wanted ✨

Help je mee hem door de Community stappen te krijgen?
[Form field description op het Community bord](https://github.com/orgs/nl-design-system/projects/29/views/1?filterQuery=form+field+description)

**Comment by eslook** [comment]
## ✨ Dit component is nu Community ✨

Help je mee hem door de Candidate stappen te krijgen?
[Form Field Description op het Candidate bord](https://github.com/orgs/nl-design-system/projects/32/views/1?filterQuery=form+field+description)

**Comment by RenateRoke** [comment]
## Candidate selectieproces: Is de component al ingesteld voor meerdere huisstijlen?

Het kernteam heeft gekeken of Form Field Description van Gemeente Utrecht al geschikt is om richting Hall of Fame te gaan. Hiervoor willen we dat de component succesvol in tenminste 2 verschillende huisstijlen is ingezet.

We hebben gekeken naar de huisstijl van:
- Gemeente Den Haag
- Gemeente Duiven
- Gemeente Haarlem
- Gemeente Hoeksche Waard
- Gemeente Hoorn
- Gemeente Nijmegen
- Provincie Zuid-Holland
- Gemeente Tilburg
- Gemeente Westervoort
- Gemeente Zevenaar
- NLDS voorbeeldthema

en we zijn tot de volgende conclusie gekomen:

🎉 Jazeker! Met jullie hulp hebben we genoeg informatie verzameld om een goed instelbaar component als Candidate beschikbaar te gaan maken.

**Comment by RenateRoke** [comment]
## Candidate selectieproces: Is de component al ingezet in meerdere projecten?

Omdat de Form Field Description van Gemeente Utrecht geschikt is voor verschillende huisstijlen heeft het kernteam ook gekeken of hij al gebruikt wordt in productie, prototypes en templates. 
Deze informatie is nodig om de component een stap verder te brengen in het Candidate proces. 

We hebben hiervoor gekeken wie de component succesvol heeft ingezet en we zijn tot de volgende conclusie gekomen:

🎉 De component wordt onder andere gebruikt door:
- Onder andere de Gemeente Den Haag gebruikt het in combinatie met Open Formulieren op https://formulieren.denhaag.nl/geboorteakte-aanvragen/stap/uw-gegevens
- Rijkshuisstijl Community gebruikt het in https://www.rijkshuisstijl-community.nl/form/
- Gemeente Purmerend gebruikt het in https://signalen-frontend-wcag.vercel.app/nl/incident

Dat maakt dat we door kunnen met de volgende stap in het selectieproces. Heb je toevoegingen aan deze lijst? We horen het graag!

**Comment by RenateRoke** [comment]
## Candidate selectieproces: Is de component al meegenomen in een WCAG-EM audit?

Om de Form Field Description van Gemeente Utrecht als een toegankelijk Candidate component beschikbaar te maken is het handig als we weten of de Community implementaties al helemaal toegankelijk kunnen worden ingezet.

We hebben hiervoor gekeken of er een toegankelijkheidsverklaring beschikbaar was voor de projecten die in productie worden gebruikt. En:

🎉 Ja! We hebben de uitkomsten van de volgende WCAG-EM audit (1 of meerdere) kunnen vinden:
- Gemeente Utrecht gebruikt het in https://toegankelijkheidsrapport.frameless.io/rapport/loket.digitaal.utrecht.nl-2024-06
- Gemeente Purmerend gebruikt het in https://toegankelijkheidsrapport.frameless.io/rapport/purmerend-signalen-2024-12
- Open Formulieren gebruikt het in https://www.horstaandemaas.nl/file/deelonderzoek-techniek-dimpact-formulieren-systeem-wcag-21-aa-pdf
- Gemeente Horst aan de Maas gebruikt het met Open Formulieren in https://www.horstaandemaas.nl/sites/horst-ad-maas/files/2023-03/WCAG%202.1%20AA%20inspectie%20-%20open-formulieren.horstaandemaas.nl%20-%201.1.pdf

### Notities voor toegankelijkheid
📝 De problemen die we hebben gevonden hebben te maken met het toegankelijk inzetten van de component, zoals beschrijvende teksten geven bij invoervelden. We gaan uiteraard goede documentatie schrijven om te helpen voorkomen dat deze toegankelijkheidsfout ontstaat.

**Comment by jeffreylauwers** [comment]
## Candidate voorbereidingsfase: Versimpeld of gesplitst

De varianten voor de Candidate component zijn bepaald, dat hebben we gedaan op basis van alle voorbeelden die we in deze Discussion hebben gezien. Als een variant veel voor komt, dan is het een algemeen nuttige variant!

### Deze varianten nemen we mee:

**Disabled** - boolean

- False (default)
- True

**Children - rich text (code) / Text  - string (design)**

### Mis je een variant?

Laat het in deze Discussion weten als er uit gebruikersonderzoek naar voren is gekomen dat deze design beslissing een hele verbetering zou zijn.

**💡 Tip**
Je kunt [varianten ook als extensie toevoegen](https://www.nldesignsystem.nl/handboek/componenten-vinden/#ik-mis-een-functionaliteit-binnen-een-candidate-of-hall-of-fame-component-wat-nu) aan een component.

**Comment by eslook** [comment]
## Candidate voorbereidingsfase: Issues bekend

We hebben gekeken bij Utrecht, Amsterdam, Den Haag, Rotterdam, Rijkshuisstijl Community, RVO en Open Formulieren naar openstaande issues en pull request gerelateerd aan de Form Field Description, de input uit de GitHub discussion meegenomen en issues bekend bij de community uitgevraagd. Daaruit zijn de volgende issues bekend gekomen.

We hebben besloten om deze issues wel mee te nemen, om de volgende redenen:

1. [Utrecht](https://github.com/nl-design-system/utrecht/)
    1. [Verwijder oude form-field-description tokens](https://github.com/nl-design-system/utrecht/issues/2862) 
    Dit komt overeen met de wens van NL Design System om de [Form Field Description](https://nldesignsystem.nl/form-field-description/) los te splitsen van de [Form Field Error Message](https://nldesignsystem.nl/form-field-error-message/) en de [Form Field Status](https://nldesignsystem.nl/form-field-status/).
1. [Open Formulieren](https://github.com/open-formulieren)
    1. [Allow setting position of description and warning](https://github.com/open-formulieren/open-forms/issues/4601)
    Dit komt overeen met wat we zien in de GitHub discussies: de positie van de Form Field Description en de Form Field Error Message kan zowel onder als boven de Form Field geplaatst worden.

We hebben besloten om deze issues niet mee te nemen, om de volgende redenen:

1. [Utrecht](https://github.com/nl-design-system/utrecht/)
    1. [feature request: align Form Field content](https://github.com/nl-design-system/utrecht/issues/2766)
    Deze issue kwam wel naar voren in de zoektocht naar Form Field Description issues, maar in de issue word aangegeven dat voor de Form Field Description geen aanpassingen nodig zijn.
    1. [feature request: add support to style error messages as an alert](https://github.com/nl-design-system/utrecht/issues/2551) 
    Deze issue kwam wel naar voren in de zoektocht naar Form Field Description issues, maar word meegenomen in de Form Field Error Message.
    1. [Should utrecht-form-field have a font-family token or not?](https://github.com/nl-design-system/utrecht/issues/2259)
    Deze issue kwam wel naar voren in de zoektocht naar Form Field Description issues, maar de specifieke wijziging voor Form Field Description is ondertussen al geimplementeerd en is dus niet relevant.
    1. [Form-field-description background-color token](https://github.com/nl-design-system/utrecht/issues/2258)
    Deze issues kwam naar voren als algemene check, omdat we vaak de `color` en `background-color` token samen beschikbaar stellen. Voor de Form Field Description is dit niet van toepassing. We hebben de issue gesloten.
    1. [alle Angular componenten in Storybook](https://github.com/nl-design-system/utrecht/issues/1588)
    Deze issue heeft geen impact op de Candidate implementatie.
    1. [alle Angular componenten getest](https://github.com/nl-design-system/utrecht/issues/1589) 
    Deze issue heeft geen impact op de Candidate implementatie.
    1. [React component library aanbieden](https://github.com/nl-design-system/utrecht/issues/833) 
    Deze issue kwam wel naar voren in de zoektocht naar Form Field Description issues, maar de genodigde aanpassing is al opgelost voor de Form Field Description.
1. [Rotterdam](https://github.com/nl-design-system/rotterdam)
    1. [Form Field-componenten maken](https://github.com/nl-design-system/rotterdam/issues/385) 
    Deze issue kwam wel naar voren in de zoektocht naar Form Field Description issues, maar de genodigde aanpassing is al opgelost voor de Form Field Description.
1. [Rijkshuisstijl Community](https://github.com/nl-design-system/rijkshuisstijl-community/)
    1. [Componenten in Storybook met nieuwe Rijkshuisstijl opleveren](https://github.com/nl-design-system/rijkshuisstijl-community/issues/2316) 
    Deze issue heeft geen impact op de Candidate implementatie omdat het specifiek gaat over het toepassen van het nieuwe Rijkshuisstijl thema.
    1. [RHC Figma Changelog](https://github.com/nl-design-system/rijkshuisstijl-community/issues/2093) 
    Deze issue kwam wel naar voren in de zoektocht naar Form Field Description issues, maar is niet van toepassing, omdat dit een verzamel issue voor de Figma Changelog betreft.
1. [RVO](https://github.com/nl-design-system/rvo/)
    1. [Documentatie: toegankelijkheid van expandable helper text](https://github.com/nl-design-system/rvo/issues/521) 
    We hebben onderzoek gedaan naar het gebruik van `details` met `summary` in de Form Field Description, en de conclusie is om het gebruik af te raden. In de issue is ook al eerder aangegeven dat hier pas iets mee gedaan zal worden als het in de GitHub discussie naar voren komt. In de estafettemodeldag van april 2026 is dit nogmaals besproken, en is er nogmaals uitgekomen dat we dit afraden.
    1. [Use existing NL Design System community React components where possible](https://github.com/nl-design-system/rvo/issues/265) 
    Deze issue kwam wel naar voren in de zoektocht naar Form Field Description issues, maar de specifieke wijziging voor Form Field Description is ondertussen al geimplementeerd en is dus niet relevant.

Uit Amsterdam en Den Haag zijn geen issues bekend gekomen.

Bekijk in [deze Candidate issue](https://github.com/nl-design-system/candidate/issues/943) hoe we tot dit resultaat zijn gekomen.

### Hoe kun je helpen om deze features wel in de Candidate implementatie te krijgen?

Pas de wijzigingen eerst in de Community implementatie toe en vraag andere organisaties in de community om deze ook in hun projecten te gebruiken. Documenteer jullie bevindingen en successen in de GitHub Discussion, deel deze in de Heartbeat en op Slack. Zo heeft het kernteam meer bewijs dat de feature wenselijk en haalbaar is en kan deze besluiten de functionaliteit alsnog, of in een volgende versie mee te nemen.

**Comment by jeffreylauwers** [comment]
## Candidate voorbereidingsfase: Anatomie bepaald

<img width="960" height="540" alt="visuele weergave van de anatomie van Form Field Description component" src="https://github.com/user-attachments/assets/c38f957e-16fb-438e-9097-a0ebb14ea14f" />

De component bestaat uit:

1. Form Field Description.

---

Deze schets is gemaakt in: [Figma](https://www.figma.com/design/0Y9Tbb373u6QGxGT6mqUnm/NL-Design-System---Doc-website---Afbeeldingen?node-id=1777-3643)

**Comment by jeffreylauwers** [comment]
## Candidate voorbereidingsfase: Design tokens bepaald

De design tokens voor de Candidate component zijn bepaald. Dat hebben we gedaan op basis van alle designs die we in deze Discussion hebben gezien. Als een beslissing vaker dan 2 keer voorkomt, is het een algemeen nuttige beslissing.

### Deze design tokens nemen we mee:

**General**
- `nl.form-field-description.color`
- `nl.form-field-description.font-family`
- `nl.form-field-description.font-size`
- `nl.form-field-description.font-weight`
- `nl.form-field-description.line-height`
- `nl.form-field-description.margin-block-end`
- `nl.form-field-description.margin-block-start`
- `nl.form-field-description.disabled.color`

### Mis je een design token?

Laat het in deze Discussion weten als er uit gebruikersonderzoek naar voren is gekomen dat deze design-beslissing een hele verbetering zou zijn.

**💡 Tip**
Je kunt design tokens ook als extensie toevoegen aan de component.

- [Voor designers: Zelf componenten uitbreiden](https://www.nldesignsystem.nl/handboek/designer/zelf-componenten-uitbreiden/)
- Voor developers: Op het moment van schrijven is de documentatie hierover nog in ontwikkeling. Vraag ons er gerust naar.
